dozer-tests/src/e2e_tests/cases/sql_aggregations/expectations.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
Do we want to add more expectations?

Reply:
Yes, let me add em, I didn't notice the other cases